tests/phpunit/tests/SVG.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+/**
+ * @coversDefaultClass \Google\Web_Stories\SVG
+ */
+class SVG extends \WP_UnitTestCase {
+	use Private_Access;
+
+	public function setUp() {
+		parent::setUp();
+		add_filter( 'web_stories_svg_already_enabled', '__return_false' );
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+		remove_filter( 'web_stories_svg_already_enabled', '__return_false' );
```

Comment:
The tests pass fine without hooking into this filter. Let's remove `setUp`/`tearDown` here.

Reply:
Removed https://github.com/google/web-stories-wp/pull/5103/commits/4b6a406a34230deb3e0347b50e29363bc098cd90